lib/usd/hdMaya/adapters/materialAdapter.cpp

```diff
@@ -105,7 +108,19 @@ void HdMayaMaterialAdapter::Populate()
     _isPopulated = true;
 }
 
-#if USD_VERSION_NUM <= 1911
+#if USD_VERSION_NUM <= 2011
+
+HdTextureResource::ID HdMayaMaterialAdapter::GetTextureResourceID(const TfToken& paramName)
+{
+    return {};
+}
+
+HdTextureResourceSharedPtr HdMayaMaterialAdapter::GetTextureResource(const SdfPath& textureShaderId)
+{
+    return {};
+}
+
+#elif USD_VERSION_NUM <= 1911
```

Comment:
this elif branch will never be entered - anything <= 1911 is by definition also <= 2011, so the first branch would be entered.

Either the order needs to be reversed (first check <= 1911, then elif <= 2011), or the first if check needs to be changed to:

```
#if USD_VERSION_NUM <= 2011 && USD_VERISON_NUM > 1911
```

Reply:
Ugh, exactly right, great catch. I'd protest that these weren't my changes, but I did review them and I completely glossed over that. :/

I'll fix these up!

Reply:
Addressed by d9ea7b2abc0f27f79638e3602c129a9812e73fd1.